bravado/testing/integration_test.py

```diff
@@ -473,6 +492,16 @@ def test_msgpack_support(self, swagger_http_server):
         assert response.headers['Content-Type'] == APP_MSGPACK
         assert unpackb(response.raw_bytes, encoding='utf-8') == API_RESPONSE
 
+    def test_redirects_are_not_followed(self, swagger_http_server):
+        response = self.http_client.request({
+            'method': 'GET',
+            'url': '{server_address}/redirect'.format(server_address=swagger_http_server),
+            'params': {},
+        }).result(timeout=1)
+
+        assert response.status_code == 301
+        assert response.headers['Location'] == '/json'
```

Comment:
The mypy report is the following
```
bravado/testing/integration_test.py:502: error: Item "Tuple[Any, ...]" of "Union[Any, IncomingResponse, Tuple[Any, IncomingResponse]]" has no attribute "status_code"
bravado/testing/integration_test.py:503: error: Item "Tuple[Any, ...]" of "Union[Any, IncomingResponse, Tuple[Any, IncomingResponse]]" has no attribute "headers"
```

What mypy is trying to say is that response is defined the have following possible types `Any`, `IncomingResponse` or `Tuple[Any, IncomingResponse]`. Not all of them (namely `Tuple[...]`) have `status_code` and `headers` attributes.

In order to fix the mypy report you might try to assert that `response` is of `IncomingResponse` type.
```
assert isinstance(response, IncomingResponse) and response.status_code == 301
```

Reply:
That seems to have done the trick. Thanks!